### PR TITLE
[Converter] Script for building the map

### PR DESCRIPTION
**Pull request by fengwuyao**
Draft a script for build map of TFJS class and TF module path.
Mapping examples: 
'GaussianDropout': 'keras.layers',
'AlphaDropout': 'keras.layers', 
'Dropout': 'keras.layers', 
'SpatialDropout1D': 'keras.layers', 
'Dense': 'keras.layers'

By using this script, we can find the corresponding module path in TF.  This is part of changes needed for supporting keras V3 format.

To see the logs from the Cloud Build CI, please join either our [discussion](https://groups.google.com/a/tensorflow.org/forum/#!forum/tfjs) or [announcement](https://groups.google.com/a/tensorflow.org/forum/#!forum/tfjs-announce) mailing list.

### CODE REVIEW

File: tfjs-converter/python/tensorflowjs/converters/build_module_map.py [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    Return:
+      RESULT_MAP[key]: the corresponding module path in TF.
+    """
+    if not RESULT_MAP:
+        raise Exception("Cannot find mapping, please build the map first.")
+    if key not in RESULT_MAP:
+        raise KeyError(f"Cannot find the module path for {key} class.")
+    return RESULT_MAP[key]
+
+
```
Comment (Linchenn) [inline tfjs-converter/python/tensorflowjs/converters/build_module_map.py:68]: You may delete extra newlines.

File: tfjs-converter/python/tensorflowjs/converters/build_module_map.py [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+        elif inspect.ismodule(obj):
+            _build_class_module_map(obj)
+
+def build_map():
+  # Build the module Map
+  _build_class_module_map(MODULE)
+  abs_path = os.path.abspath(__file__)
+  root_path = os.path.dirname(os.path.dirname(os.path.dirname(abs_path)))
+  path = os.path.join(root_path, 'tfjs-layers')
+  _build_ts_class_module_map(path)
```
Comment (Linchenn) [inline tfjs-converter/python/tensorflowjs/converters/build_module_map.py:52]: The RESULT_MAP might not be needed. If a class name does not exist in Keras, it should return null. If a class name does not exist in TFJS, you will not have a chance to call `get_module_path`. For both cases, TFCLASS_MODULE_MAP is enough.

File: tfjs-converter/python/tensorflowjs/converters/build_module_map.py [hunk 3]
```diff
@@ -0,0 +1,74 @@
+import tensorflow.keras as keras
```
Comment (Linchenn) [inline tfjs-converter/python/tensorflowjs/converters/build_module_map.py:1]: Add the license.

**Review by Linchenn — CHANGES_REQUESTED** [review]
You may also do the tests for TFCLASS_MODULE_MAP.

File: tfjs-converter/python/tensorflowjs/converters/tf_module_mapper.py [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+        if inspect.isclass(obj):
+            # Retrive the module path from tensorflow.
+            parts = str(tf_export.get_canonical_name_for_symbol(obj, api_name='keras')).split(".")
+            # Map the class name with module path exclude the class name.
+            TFCLASS_MODULE_MAP[name] = ".".join(parts[:-1])
+
+        elif inspect.ismodule(obj):
+            _build_class_module_map(obj)
+
+def build_map():
```
Comment (Linchenn) [inline tfjs-converter/python/tensorflowjs/converters/tf_module_mapper.py:40]: get_module_path and others could call _build_class_module_map directly

File: tfjs-converter/python/tensorflowjs/converters/tf_module_mapper.py [hunk 5]
```diff
@@ -0,0 +1,55 @@
+# Copyright 2018 Google LLC
```
Comment (Linchenn) [inline tfjs-converter/python/tensorflowjs/converters/tf_module_mapper.py:1]: We could update to 2023.

**Review by Linchenn — APPROVED** [review]
LGTM! Just some nits. Thanks!

**Review by mattsoulanille — APPROVED** [review]
LGTM
